src/devices/multir.ts

```diff
@@ -0,0 +1,118 @@
+import * as fz from "../converters/fromZigbee";
+import * as exposes from "../lib/exposes";
+import * as m from "../lib/modernExtend";
+import * as reporting from "../lib/reporting";
+import type {DefinitionWithExtend, Fz} from "../lib/types";
+
+const e = exposes.presets;
+
+const fzLocal = {
+    MIRSO100: {
+        cluster: "ssIasZone",
+        type: "raw",
+        convert: (model, msg, publish, options, meta) => {
+            switch (msg.data[3]) {
+                case 0:
+                    return {action: "single"};
+                case 1:
+                    return {action: "double"};
+                case 128:
+                    return {action: "hold"};
+            }
+        },
+    } satisfies Fz.Converter<"ssIasZone", undefined, "raw">,
+};
+
+export const definitions: DefinitionWithExtend[] = [
+    {
+        zigbeeModel: ["MIR-MC100"],
+        model: "MultIR doors sensor",
```

Comment:
```suggestion
        model: "MIR-MC100",
```

This seems like a better model (same for others)

Reply:
Has finished modifying